packages/tree-widget/src/components/trees/TreeWithRuleset.tsx

```diff
@@ -0,0 +1,172 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) 2019 Bentley Systems, Incorporated. All rights reserved.
+ * Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+ * Based on unified-navigator from Bentley.
+ * See https://bentleycs.visualstudio.com/iModelTechnologies/_git/unified-navigator
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import { RegisteredRuleset, Ruleset } from "@bentley/presentation-common";
+import {
+  IPresentationTreeDataProvider,
+  usePresentationTreeNodeLoader,
+  useUnifiedSelectionTreeEventHandler,
+  PresentationTreeDataProvider,
+} from "@bentley/presentation-components";
+import {
+  useVisibleTreeNodes,
+  SelectionMode,
+  ControlledTree,
+} from "@bentley/ui-components";
+import { Presentation } from "@bentley/presentation-frontend";
+import "./TreeWithRulesetTree.scss";
+import { connectIModelConnection } from "@bentley/ui-framework";
+import { IModelConnection } from "@bentley/imodeljs-frontend";
+
+/**
+ * Properties for the [[ControlledSpatialContainmentTree]] component
+ * @internal
+ */
+export interface ControlledTreeProps {
+  iModel: IModelConnection;
+  dataProvider: IPresentationTreeDataProvider;
+  rulesetId: string;
+  pageSize?: number;
+}
+
+/**
+ * Properties for the [[TreeWithRulesetTree]] component
+ * @alpha
```

Comment:
Correct me if im wrong @calebmshafer, I don't believe we're publishing API docs like core does, so i think all `alpha` jsdocs can be removed.